src/sentry/static/sentry/app/views/performance/realUserMonitoring/measurementsHistogramQuery.tsx

```diff
@@ -41,12 +42,22 @@ type Props = RequestProps & {
 };
 
 function getMeasurementsHistogramRequestPayload(props: any) {
-  const {measurements, numBuckets, min, max, dataFilter, eventView, location} = props;
+  const {
+    measurements,
+    numBuckets,
+    min,
+    max,
+    precision,
+    dataFilter,
+    eventView,
+    location,
+  } = props;
   const baseApiPayload = {
     measurement: measurements,
-    num_buckets: numBuckets,
+    numBuckets,
```

Comment:
Do we prefer camelcase or snake case for our serialization? I've seen both 🤔

Reply:
ideally camelCase, but mistakes have been made.